components/AuthButton.server.js

```diff
@@ -1,21 +1,21 @@
-import React from 'react'
+import React, { useReducer } from 'react'
```

Comment:
Is this needed?

Reply:
Looks like an accidental vscode auto import. Thanks for finding! :)